ros2node/package.xml

```diff
@@ -20,6 +20,7 @@
   <depend>ros2cli</depend>
```

Comment:
```suggestion
  <exec_depend>ros2cli</exec_depend>
```

Reply:
I believe `<depend>` is correct here. All other ros2* packages have it as well, I guess the main reason is to register the command verbs with `ros2cli` (setup.py)?

Reply:
It's not necessary for that.  There is never a reason to use `<depend>` in a pure Python package; it has no build step.

Reply:
OK I see. Is it OK for you to leave it as is, since the change would be somewhat unrelated to this PR?